### PR TITLE
Add link to helper article in version control notice

### PR DESCRIPTION
**Pull request by mike-day**
## Tasks

- [X] I have signed a [Contributor License Agreement (CLA)](https://github.com/studiopress/pattern-manager/wiki/For-Developers#contributor-license-agreement-cla) with WP Engine.

### Summary of changes

This PR adds a link to a new [helper article](https://developer.wpengine.com/knowledge-base/using-git-with-a-wordpress-theme/) about setting up version control for a WordPress theme.

### How to test

1. Switch to a theme that has not had the version control notice dismissed (or delete the`patternmanager_version_control_notice_dismissed_themes` DB entry from usermeta)
2. Confirm that a link is displayed in the version control notice
3. Confirm that the link works

### Notes & Screenshots

Screenshot 2023-06-02 at 9 33 15 AM

### CODE REVIEW

**Comment by mike-day** [comment]
Is it preferable for the link to open in a new tab? Currently, the article opens in the same tab.

**Comment by kienstra** [comment]
>Is it preferable for the link to open in a new tab? Currently, the article opens in the same tab.

Yeah, that'd probably be better. It's to an external URL.

File: wp-modules/app/js/src/components/VersionControlNotice/index.tsx [hunk 1]
```diff
@@ -19,9 +20,23 @@ export default function VersionControlNotice( {
 			status="warning"
 			onRemove={ handleDismiss }
 		>
-			{ __(
-				'No version control detected for this theme. We recommend adding version control so you do not lose your patterns during theme updates.',
-				'pattern-manager'
+			{ createInterpolateElement(
```
Comment (kienstra) [inline wp-modules/app/js/src/components/VersionControlNotice/index.tsx:23]: Great use of `createInterpolateElement()`

**Review by kienstra — APPROVED** [review]
Hi @mike-day,
This looks really good, nice `createInterpolateElement()` use.

Screenshot 2023-06-02 at 2 57 54 PM

**Comment by mike-day** [comment]
> > Is it preferable for the link to open in a new tab? Currently, the article opens in the same tab.
> 
> Yeah, that'd probably be better. It's to an external URL.

This is covered with [e203a67](https://github.com/studiopress/pattern-manager/pull/190/commits/e203a6776a1fb5cef7ee7d494f31098947692236).

**Pull request merged**